lit/v5.2.0.lit

```diff
@@ -0,0 +1,111 @@
+\use-plugin{concourse-docs}
+
+\release-version{5.2.0}
+\release-date{2019-5-XX}
+
+\note{fix,breaking}{
+  Change flag for \code{fly builds --teams} to \code{-n} from \code{-t} to make
+  it consistent across \code{fly}. Thanks to a PR by \ghuser{stigtermichiel}.
+}
+
+\note{feature}{
+  As We move toward adding more RBAC roles, a new pipeline-operator role was
+  added (thanks to \ghuser{tlwr} and \ghuser{paroxp}) which is between viewer
+  and member and can:
+
+  - AbortBuild
```

Comment:
This should be formatted like so:

```
\list{
  AbortBuild
}{
  CreateJobBuild
}
```

(...it's a bit verbose)

For extra flair you could do `\code{AbortBuild}` and such.